contracts/README.md

```diff
@@ -40,14 +40,14 @@ The Smart Contracts can be installed separately from the other components of the
 populus compile
 
 # tests
-pytest
-pytest -p no:warnings -s
-pytest tests/test_uraiden.py -p no:warnings -s
+py.test
```

Comment:
`pytest` also works fine

Reply:
ah, you are right. I was confused because `pytest` would use my global installation and `py.test` the local one from the virtualenv. But `py.test` is actually being deprecated in favor of `pytest`...